src/main/resources/config.yml

```diff
@@ -8,6 +8,7 @@ options:
   backwards-compatibility: false
   chat-prefix: '&a&lSlimefun 4&7> '
   armor-update-interval: 10
+  radiation-update-interval: 1
```

Comment:
Probably a bit too fast for a default value in my opinion

Reply:
After all... we do item comparisons for EVERY item in the player inventory on every tick here O.o

Reply:
In seconds.
And while we are on it, `armor-update-interval` applies every 10 seconds so it takes 10 seconds for a hazmat suit to apply.